lib/chart/model/chart_state.dart

```diff
@@ -189,25 +193,28 @@ class ChartState<T> {
   /// [ItemOptions].
   ///
   /// If you need more customization of the individual chart items see [_widgetItemRenderer]
-  static ChartDataRendererFactory<T?> _defaultItemRenderer<T>(ItemOptions itemOptions) {
+  static ChartDataRendererFactory<T?> _defaultItemRenderer<T>(
+      ItemOptions itemOptions) {
     return (chartState) {
       return ChartLinearDataRenderer<T?>(
-        chartState,
-        chartState.data.items
-            .mapIndexed(
-              (lineKey, items) => items
-                  .mapIndexed((itemKey, item) => LeafChartItemRenderer(
-                        item,
-                        chartState.data,
-                        itemOptions,
-                        itemKey: itemKey,
-                        listKey: lineKey,
-                        drawDataItem: itemOptions.itemBuilder(ItemBuilderData<T?>(item, itemKey, lineKey)) as DrawDataItem,
-                      ))
-                  .toList(),
-            )
-            .expand((element) => element)
-            .toList());
+          chartState,
+          chartState.data.items
+              .mapIndexed(
+                (lineKey, items) => items
+                    .mapIndexed((itemKey, item) => LeafChartItemRenderer(
+                          item,
+                          chartState.data,
+                          itemOptions,
+                          itemKey: itemKey,
+                          listKey: lineKey,
+                          drawDataItem: itemOptions.itemBuilder(
+                                  ItemBuilderData<T?>(item, itemKey, lineKey))
+                              as DrawDataItem,
+                        ))
+                    .toList(),
+              )
+              .expand((element) => element)
```

Comment:
What does this do?

Reply:
This is just a file change from `dart format .`
But this is Factory for renderers (ChartLinearDataRenderer).

If you wander about `.expand` it's just merging Chart items from: `List<List<ChartItem>>` to `List<ChartItem>` and passing them to a renderer, (Renderer will know listKey and itemKey so there is no need for List<List<>> structure).